1danko/sap-fioneer-qa-challenge
# [QA Automation Challenge] SAP Fioneer

### Tasks to Complete
The automation should consist of the following 3 tests:
**Test 1**
_1. Go to page SAP Fioneer | World-class software solutions for financial services
2. Verify if the following bookmarks are visible – Banking, Insurance, Finance & ESG, Services, Partners, Company, Resources_
**Test 2**
_1. Go to page SAP Fioneer | World-class software solutions for financial services
2. On the Finance & ESG bookmark, click on Financial Control
3. Verify if a user has been redirected to the correct page: SAP Fioneer | Financial Control | Optimise accounting & finance_
**Test 3**
_1. Go to page SAP Fioneer | World-class software solutions for financial services
2. Click on Get in touch button
3. Verify if a user has been redirected to the correct page: SAP Fioneer | Contact | Get in touch!
4. On empty contact form click Submit_
